### PR TITLE
translate (docs/pt/guidelines/content/set-plagiarism.md)

### PR DESCRIPTION
**Pull request by Arthur-Barreto**
translate `docs/pt/guidelines/content/set-plagiarism.md` to pt-br

### CODE REVIEW

File: docs/pt/guidelines/content/set-plagiarism.md
Comment (wescopeland) [inline docs/pt/guidelines/content/set-plagiarism.md:1]: The term "badge" is used quite frequently. Is this consistent with our other pt-BR translations?

Similarly, "Leaderboards", "Rich Presence", and "In-Game" are also kept in English.

Comment (Arthur-Barreto) [reply docs/pt/guidelines/content/set-plagiarism.md:1]: ok, going to keep them on translation

Comment (Arthur-Barreto) [reply docs/pt/guidelines/content/set-plagiarism.md:1]: About the first question, yes, it's consistent, sorry about not answering that

**Comment by Arthur-Barreto** [comment]
@wescopeland I've updated what you have asked me

**Comment by wescopeland** [comment]
Hey @Arthur-Barreto -- thanks for your contributions. I'll do another review round this evening.

**Review by wescopeland — APPROVED** [review]

**Pull request merged**